README.md

```diff
@@ -161,70 +191,109 @@ You must download `envsubst` for your platform from [https://github.com/a8m/envs
     In your terminal, export (or SET for Windows) your top level domain
     and the backend you are using. 
     
-    For example for domain `sockshop.mycompany.com` and `memory` backend, use the following 
+    For example for domain `sockshop.mycompany.com` and `core` backend, use the following 
     
     ```bash
     $ export SOCKSHOP_DOMAIN=sockshop.mycompany.com
-    $ export SOCKSHOP_BACKEND=memory                            
+    $ export SOCKSHOP_BACKEND=core                            
     
-    $ envsubst -i k8s/optional/ingress.yaml| kubectl apply -f -
+    $ envsubst -i k8s/optional/ingress.yaml | kubectl apply -n sockshop-${SOCKSHOP_BACKEND} -f -
     
-    $ kubectl get ingress  
+    $ kubectl get ingress -n sockshop-${SOCKSHOP_BACKEND}  
     
-    NAME               HOSTS                                                                                         ADDRESS           PORTS   AGE
-    mp-ingress         mp.memory.mycompany.com                                                                       XXX.XXX.XXX.XXX   80      12d
-    sockshop-ingress   memory.sockshop.mycompany.com,jaeger.memory.sockshop.mycompany.com,api.memory.mycompany.com   XXX.XXX.XXX.XXX   80      12d
+    NAME               HOSTS                                                                                            ADDRESS           PORTS   AGE
+    mp-ingress         mp.core.sockshop.mycompany.com,swagger.core.sockshop.mycompany.com                               XXX.XXX.XXX.XXX   80      12d
```

Comment:
You should have just `mp.core.sockshop.mycompany.com` host here. See comments in `ingress.yaml` why.